WooCommerce/Classes/POS/Presentation/ItemRowView.swift

```diff
@@ -17,7 +17,6 @@ struct ItemRowView: View {
                 ProductImageThumbnail(productImageURL: URL(string: imageSource),
                                       productImageSize: Constants.productCardHeight,
                                       scale: scale,
-                                      productImageCornerRadius: Constants.productCardCornerRadius,
```

Comment:
We do not need corner radius on the product image, we have one for the whole view